.claude/hooks/file-validator.py

```
@@ -0,0 +1,232 @@
+        '*.p12',
+        '*.jks'
+    ]
+    
+    # Check if file matches any sensitive pattern
+    for pattern in sensitive_patterns:
+        if pattern in file_path.lower():
+            return True, pattern
+            
+    return False, None
+
+def is_allowed_exception(file_path, tool_name):
```

Ruff (0.12.2)

Unused function argument: `tool_name`

(ARG001)